src/components/vaForm/template.test.tsx

```diff
@@ -62,7 +63,9 @@ describe('VaForm Component', () => {
     test('renders form title and basic information', () => {
       const { container } = render(<VaForm {...formattedMockData} />)
 
-      expect(screen.getByText('About VA Form 21-0781')).toBeInTheDocument()
+      expect(
+        screen.getByRole('heading', { level: 1, name: 'About VA Form 21-0781' })
```

Comment:
Unrelated to this change. I just saw an opportunity to make this test a little better.